pkg/cli/root.go

```diff
@@ -26,7 +26,7 @@ func GetCommand() *cobra.Command {
 
 	cmd.AddCommand(getGetCommand())
 	cmd.AddCommand(getAddCommand())
-	//cmd.AddCommand(getRemoveCommand())
```

Comment:
why is this removed ?

Reply:
it had been commented out because you could not remove (unload) a plugin. I was removing the commented text here